packages/storybook/cxl-ui/cxl-vaadin-accordion.stories.js

```diff
@@ -18,10 +19,12 @@ Object.assign(CXLPlaybookAccordion, {
 CXLVaadinAccordionThemeArchive.storyName = 'cxl-accordion-card';
 CXLVaadinAccordionThemeFaq.storyName = '[theme=cxl-faq]';
 CXLVaadinAccordionThemeMinidegreeTrack.storyName = '[theme=cxl-minidegree-track]';
+CXLHubsAndPlaybooks.storyName = 'cxl-hubs-and-playbooks';
```

Comment:
Existing stories are either

* element names
* element themes

`cxl-hubs-and-playbooks` needs to become one of the above, not a random invention.

It's seems likely you're looking to create a new cxl-accordion-card theme, not a whole new element.

Reply:
@lkraav This supposed to be a story with hubs, and playbooks in one place 
@heshfekry described it here: https://app.clickup.com/t/wkyr89, 
yet I'm still not sure how to name this `theme`. 
`[theme=hubs-and-playbooks]` is good enough?